src/libraries/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -322,24 +322,6 @@ public abstract partial class TransportContext
         protected TransportContext() { }
         public abstract System.Security.Authentication.ExtendedProtection.ChannelBinding? GetChannelBinding(System.Security.Authentication.ExtendedProtection.ChannelBindingKind kind);
     }
-    public enum NetworkError : int
```

Comment:
Why're this being moved?

Reply:
NetworkError specifically? It's for use with NetworkException.

Reply:
I meant both. I guess those are specific to Connections but were added to System.Net.Primitives as they are primitives, but now that we're not exposing connections we don't want them here?

Reply:
We added NetworkException as part of the Connections work. It's used by Connections APIs.

But we also modified NetworkStream to throw NetworkException instead of IOException, to support Connections scenarios involving NetworkStream. That's why it's in System.Net.Primitives instead of just in System.Net.Connections.

This PR reverts the NetworkStream changes and moves NetworkException into the Connections code.

Reply:
I see makes sense. Thanks for elaborating.